eval/run.rkt

```diff
@@ -44,10 +44,17 @@
         [n (in-naturals varc)]
         [precision (in-vector precisions)]
         [repeat (in-vector repeats)]
-        #:unless (and (not first-iter?) repeat))
+        [hint (if vhint
+                  (in-vector vhint)
+                  (in-producer (const #t)))]
```

Comment:
If no `vhint` is provided then execute all the instructions and treat local `hint` as `#t`